tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -3181,31 +3181,15 @@ def test_is_metrics_extracted_data_is_included(self) -> None:
         response = self._make_on_demand_request(params)
         self._assert_on_demand_response(response)
 
-    def test_transaction_user_misery(self) -> None:
+    def test_user_misery_uses_alias(self) -> None:
+        """Verify that user_misery(300) is used instead of user_misery_300"""
         user_misery_field = "user_misery(300)"
-        apdex_field = "apdex(300)"
-        params = {
-            "field": [user_misery_field, apdex_field],
-            "project": self.project.id,
-            "query": "",
-        }
-        specs = self._create_specs(params, groupbys=["transaction"])
-        for spec in specs:
-            self.store_on_demand_metric(1, spec=spec)
+        params = {"field": [user_misery_field], "project": self.project.id, "query": ""}
+        self._create_specs(params)
         response = self._make_on_demand_request(params)
         self._assert_on_demand_response(response, expected_on_demand_query=True)
-        assert response.data == {
-            "data": [{user_misery_field: 0.0, apdex_field: 0.0}],
-            "meta": {
-                "fields": {user_misery_field: "number", apdex_field: "number"},
-                "units": {user_misery_field: None, apdex_field: None},
-                "isMetricsData": True,
-                "isMetricsExtractedData": True,
-                "tips": {},
-                "datasetReason": "unchanged",
-                "dataset": "metricsEnhanced",
-            },
-        }
+        assert response.data["meta"]["fields"] == {user_misery_field: "number"}
+        assert response.data["meta"]["units"] == {user_misery_field: None}
```

Comment:
Is there another on-demand user_misery test covering it actually working? It's a bit of a tricky field, we should assert the user_misery value !=0 for on-demand in events and events-stats at least once.

Reply:
We know it falls back to Discover, thus, we would need to create some events rather than metrics.

We have this original test but we may want to review it to see if it's given us the value we want:
https://github.com/getsentry/sentry/pull/58011/files#diff-de739774ae860e66c99b25f087ec8b0c7768ca3fa65a6da26cf9794e2de7e2dcR2323